javaagent/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ dependencies {
     //spring modules
     implementation("org.springframework.boot:spring-boot-starter-web")
 
-    agent("io.opentelemetry.javaagent:opentelemetry-javaagent:2.8.0")
+    agent("io.opentelemetry.javaagent:opentelemetry-javaagent:2.9.0-SNAPSHOT")
```

Comment:
Don't merge until next release of `opentelemetry-java-instrumentation` and the `-SNAPSHOT` is reverted.

Reply:
Is it that important tho for examples? As long as renovate knows to bump it.....I dunno.